### PR TITLE
Added a New Functionality

### LINKED ISSUE #271: Add a Functionality
I am working on it.

### PR DESCRIPTION
**Pull request by naveentherocker**
Fixes #271 
Added a New Functionality to the Desert Theme along with required Floor Tiles.

Second Step #206
Added Separate Messages for Other Tiles
 
You can view the changes as soon as #161 got fixed.

Ping @niccokunzmann @fossasia/labyrinth-admins

### CODE REVIEW

**Comment by naveentherocker** [comment]
@xeon-zolt Please see this

File: tiles/rooms/floor/camel.svg [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+     id="path26"
+     inkscape:connector-curvature="0" /><image
+     y="149.93457"
+     x="164.35454"
+     id="image47"
+     xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAAA8CAYAAADxJz2MAAAACXBIWXMAAAsTAAALEwEAmpwYAAAK
+T2lDQ1BQaG90b3Nob3AgSUNDIHByb2ZpbGUAAHjanVNnVFPpFj333vRCS4iAlEtvUhUIIFJCi4AU
+kSYqIQkQSoghodkVUcERRUUEG8igiAOOjoCMFVEsDIoK2AfkIaKOg6OIisr74Xuja9a89+bN/rXX
+Pues852zzwfACAyWSDNRNYAMqUIeEeCDx8TG4eQuQIEKJHAAEAizZCFz/SMBAPh+PDwrIsAHvgAB
```
Comment (niccokunzmann) [inline tiles/rooms/floor/camel.svg]: This is not an svg camel. You can use the svg camel from flappy SVG, I think.

**Comment by niccokunzmann** [comment]
You improved a lot of other elements, thanks for that, too!

**Comment by naveentherocker** [comment]
Thank You for the merge and you are welcome

**Pull request merged**